resources/qml/TimelineRow.qml

```diff
@@ -26,6 +26,7 @@ AbstractButton {
     required property string filesize
     required property string url
     required property string thumbnailUrl
+    required property string fullyReadEventId
```

Comment:
I don't think you need to pass this property through. Can't you just do chat.model.fullyReadEventId at the usage site?